src/dolphin/workflows/displacement.py

```diff
@@ -164,6 +173,33 @@ def run(
         mask_file=cfg.mask_file,
     )
 
+    # ##############################################
+    # 4. Estimate corrections for each interferogram
+    # ##############################################
+
+    if cfg.correction_options.dem_file is None:
+        raise ValueError(
+            "DEM file is not given, skip estimating tropospheric corrections..."
+        )
```

Comment:
As you pointed out, we could make this a warning.
I was trying to decide if there was a better way to flag the tropo correction on/off, but i haven't come up with anything yet